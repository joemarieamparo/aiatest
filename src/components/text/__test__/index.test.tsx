import React from 'react'
import {render} from '@testing-library/react-native'
import Text from '../index'

const createTestProps = (propsData: Object) => ({
    ...propsData
  })
describe('Screen: Text', () => {
    it('Text: renders correctly', () => {
        const props: any = createTestProps({
          testId: 'id',
          text: 'Text',
          style: {}
        })
        const {toJSON} = render(<Text {...props} />)
        expect(toJSON()).toMatchSnapshot();
    })
})