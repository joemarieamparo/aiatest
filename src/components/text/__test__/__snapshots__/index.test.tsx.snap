// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen: Text Text: renders correctly 1`] = `
<Text
  allowFontScaling={false}
  style={Object {}}
  testID="text_id"
>
  Text
</Text>
`;
